config/dev-amo.js

```diff
@@ -24,4 +24,6 @@ module.exports = {
       ],
     },
   },
+
+  enableCollectionEdit: true,
```

Comment:
It might be nice to set this flag as default to on in `development.js` as well... it feels like it should be enabled in development mode no matter what.